scanner/services/matcher.go

```diff
@@ -112,8 +123,11 @@ func (s *matcherService) RegisterServiceServer(grpcServer *grpc.Server) {
 
 // AuthFuncOverride specifies the auth criteria for this API.
 func (s *matcherService) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
-	// TODO: Setup permissions for matcher.
-	return ctx, allow.Anonymous().Authorized(ctx, fullMethodName)
+	auth := matcherAuth
+	if !buildinfo.ReleaseBuild {
+		auth = allow.Anonymous()
+	}
```

Comment:
nit: let's also leave a comment in case people question why this is done, something along the lines of the test advantages it brings (and it being harder than "normal" central API services).